zebra/zebra_nb_config.c

```diff
@@ -1550,12 +1550,14 @@ int lib_route_map_entry_set_action_source_v4_modify(
 			if (pif != NULL)
 				break;
 		}
-		if (pif == NULL) {
-			snprintf(args->errmsg, args->errmsg_len,
-				 "is not a local address: %s",
-				 yang_dnode_get_string(args->dnode, NULL));
-			return NB_ERR_VALIDATION;
```

Comment:
We will fill the errmsg still and return to main callback function. Infra is not ready for that and can be done in seperate PR after discussion on Yang channel. For time being this looks good.